### PR TITLE
enhance CI E2E test

### PR DESCRIPTION
**Pull request by mengfei25**
CI E2E test:
1. Full huggingface BF16 training accuracy
2. Full huggingface FP16 training accuracy
3. 16 timm_models BF16 training accuracy
4. 16 torchbench BF16 training accuracy

Nightly E2E test:
1. Full huggingface accuracy
2. Full torchbench BF16 training accuracy
3. Full timm_models FP16 training accuracy

### CODE REVIEW

File: .github/scripts/apply_torch_pr.py [hunk 1]
```diff
@@ -7,17 +7,27 @@
 
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--pr-list', '-n', nargs='+', default=[], required=True)
-parser.add_argument('--repo', '-r', default='https://github.com/pytorch/pytorch.git', type=str)
+parser.add_argument('--pr-list', '-n', nargs='+',
+    default=[
+        #Upgrade submodule oneDNN to v3.4
+        "https://github.com/pytorch/pytorch/pull/122472",
```
Comment (chuanqi129) [inline .github/scripts/apply_torch_pr.py:13]: Can we keep previously PR apply logic and maintainer this standalone patch into `.github/script`?

**Review by chuanqi129 — APPROVED** [review]

**Comment by chuanqi129** [comment]
Hi @mengfei25 @fengyuan14 there are a lot hf model accuracy failures caused by eager 2 run differ, please help to double check it.

**Comment by mengfei25** [comment]
> Hi @mengfei25 @fengyuan14 there are a lot hf model accuracy failures caused by eager 2 run differ https://github.com/intel/torch-xpu-ops/actions/runs/9075104236/job/24935078688?pr=216, please help to double check it.

oneDNN patch missed

**Comment by mengfei25** [comment]
CI UT: 44 min
CI E2E: 1h 43 min

**Review by EikanWang — APPROVED** [review]

**Pull request merged**